README.md

```diff
@@ -8,7 +8,7 @@ First install all required dependencies. See https://jekyllrb.com/ for getting s
 Then run command
 
 ```
-bundle exec jekyll serve
+bundle exec jekyll serve --baseurl=""
```

Comment:
this is not enough here.
You said we had to comment out some stuff in the gemfile ?
Let's go from the principal that the person using this is new. 
How can we make it work locally?